optaplanner-quickstarts/quarkus-school-timetabling/pom.xml

```diff
@@ -13,6 +13,11 @@
     <version.org.optaplanner>7.43.0-SNAPSHOT</version.org.optaplanner>
     <version.io.quarkus>1.7.0.Final</version.io.quarkus>
 
+    <version.org.webjars.bootstrap>4.3.1</version.org.webjars.bootstrap>
+    <version.org.webjars.jquery>3.4.1</version.org.webjars.jquery>
+    <version.org.webjars.font-awesome>5.11.2</version.org.webjars.font-awesome>
+    <version.org.webjars.momentjs>2.24.0</version.org.webjars.momentjs>
+
```

Comment:
fyi there were already merged this PR https://github.com/kiegroup/optaplanner/pull/898 which handles the movement of hardcoded dependencyversions to version property, so just rebase this PR

Reply:
@mareknovotny it was done only for `optaplanner-quickstarts/spring-boot-school-timetabling` so I decided to apply same pattern for quarkus one